src/App.js

```diff
@@ -1,7 +1,43 @@
-import './App.css';
+import React, { useEffect, useState } from "react";
+
+import Authorization from "./components/Authorization/Authorization";
+import WeatherBlockWrap from "./components/WeatherBlock/WeatherBlockWrap";
+
+import { checkAuthorization } from "./service/auth/auth.service";
+
+import style from "./app.module.scss";
 
 const App = () => {
-  return <div />
-}
+  const [isAuthorized, setIsAuthorized] = useState(false);
```

Comment:
isAuthorized is used in every component, it is better to use some store managment tool, like redux (or zustand)